src/main/java/com/genesys/workspace/VoiceApi.java

```diff
@@ -227,8 +254,16 @@ private void onCallStateChanged(Map<String, Object> data) {
         call.setCallUuid(callUuid);
         call.setParticipants(participants);
         call.setUserData(userData);
-        call.setCapabilities(Arrays.stream(capabilities).map(v -> Capability.fromString((String)v)).filter(Objects::nonNull).toArray(n -> new Capability[n]));
+        List<Capability> capabilityList = new ArrayList<>(10);
+        for(Object capability: capabilities){
 
+            if(capability != null){
```

Comment:
broken logic, the code should filter not null objects after conversion fromString